apis/r/tools/install-tiledb-r.R

```diff
@@ -42,4 +42,13 @@ ctb <- contrib.url(getOption("repos"))
 names(ctb) <- getOption("repos")
 dbr <- db[idx[valid], 'Repository']
 (repos <- names(ctb[ctb %in% dbr]))
+
+# BSPM doesn't respect `repos`
```

Comment:
As a long-time user of bspm and partial contributor to it I think that is not a correct way of stating it.

After all bspm (and its cousin rspm using the p3m.dev binaries not packaged for apt, dnf, ...) is designed to use apt and dnf and alike.  It's simply a different approach.   I am fairly certain bspm does available.packages and hence repos but then has to reconcile different 'sets' of packages.  There are some options that control its behavior, and its author added more.

Reply:
If there's an option to tell bspm to use r-universe of APT, then I'm all for using that. However, with bspm enabled and passing repos that doesn't contain CRAN à la
```
install.packages("tiledb", repos = "https://tiledb-inc.r-universe.dev")
```
We get tiledb-r from APT, not from R-universe

Reply:
I am kinda bored by this because we go over all of this every three or four weeks.  Do whatever you need to do it is all fine by me.